### PR TITLE
"Rollout and activate" action

### PR DESCRIPTION
**Pull request by SkoryUladzislau**
The ability to rollout and publish selected page in one click has been implemented. By means of an additional parameter in the request - shouldActivate. The servlet understands whether the page should be activated or not. A new service has been added that, in a multi-threaded pool, calls page replication if necessary.

### CODE REVIEW

File: core/src/main/java/com/exadel/etoolbox/rolloutmanager/core/services/impl/PageReplicationServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+            LOG.warn("Replication failed - target page is null, page path: {}", targetPath);
+            return status;
+        }
+        try {
+            replicator.replicate(session, ReplicationActionType.ACTIVATE, targetPath);
+        } catch (ReplicationException ex) {
+            status.setSuccess(false);
+            LOG.error("Exception during page replication", ex);
+        }
+        status.setSuccess(true);
```
Comment (smiakchilo) [inline core/src/main/java/com/exadel/etoolbox/rolloutmanager/core/services/impl/PageReplicationServiceImpl.java:115]: ❗ Need to move this inside `try`, or else the status success will be reset to "true" after set to "false"

**Review by smiakchilo — APPROVED** [review]
Please add the "Licence" headers to the files where they are missing

File: core/src/main/java/com/exadel/etoolbox/rolloutmanager/core/services/impl/PageReplicationServiceImpl.java (hunk 1 above)
Comment (AlKaliada) [reply core/src/main/java/com/exadel/etoolbox/rolloutmanager/core/services/impl/PageReplicationServiceImpl.java]: done

**Review by AlKaliada — APPROVED** [review]

**Comment by AlKaliada** [comment]
I have read the CLA Document and I hereby sign the CLA

**Pull request merged**